backend/open_webui/main.py

```
@@ -1470,6 +1470,140 @@ async def chat_completed(
         )
 
 
+# Get attestation report of intel quote and nvidia payload
+@app.get("/api/attestation/report")
+async def chat_attestation(
+    request: Request, model: str, user=Depends(get_verified_user)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)